vmm/Cargo.toml

```diff
@@ -20,7 +20,7 @@ anyhow = "1.0"
 arch = { path = "../arch" }
 devices = { path = "../devices" }
 epoll = ">=4.0.1"
-kvm-bindings = { git = "https://github.com/cloud-hypervisor/kvm-bindings", branch = "ch", features = ["with-serde", "fam-wrappers"] }
+kvm-bindings = { git = "https://github.com/cloud-hypervisor/kvm-bindings", branch = "ch", features = ["fam-wrappers"] }
```

Comment:
Now the problem is: `"with-serde"` is needed on x86_64, but not on aarch64. 
So don't modify this, so x86_64 should build.
For aarch64, replace https://github.com/cloud-hypervisor/cloud-hypervisor/blob/master/.github/workflows/cross-build.yaml#L26 with `run: sed -i 's/"with-serde",\ //g' vmm/Cargo.toml && sed -i 's/"with-serde",\ //g' hypervisor/Cargo.toml` in your PR. Then both x86_64 and  aarch64 should build.